doc/tutorials/java-kdb.md

```diff
@@ -85,34 +85,51 @@ The _user_ namespace is accessible without special rights, but if you try to wri
 
 ### Traversing Keys in a `KeySet`
 
+First we create a new `KDB` session and fetch all keys for the desired namespace, in this example the whole `user` namespace. Since all all keys are put in the passed `keySet` variable we can then iterate through it.
+The `at(int)` method returns a new `Key` object for the native key with the corresponding position within the `keySet`.
```

Comment:
new `Key` object sounds like that something new is created but this isn't the case, is it?

Reply:
Actually there is a new `Key` object (Java representation for the native key) created.